proofpoint_tap/plugin.spec.yaml

```diff
@@ -78,6 +78,11 @@ types:
       description: Header reply to
       type: string
       required: false
+    message_guid:
```

Comment:
I don't see this used in code anywhere?

Reply:
proofpoint_tap/komand_proofpoint_tap/util/tap_formatter.py  has it added.  The value will be extracted from the HTML just like the other values in that clean_data hash in tap_formatter.py